project/AppDependencies.scala

```diff
@@ -2,21 +2,37 @@ import sbt.*
 
 object AppDependencies {
 
-  private val bootstrapVersion = "8.6.0"
+  private val bootstrapVersion = "9.1.0"
 
   val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
     "uk.gov.hmrc" %% "bootstrap-frontend-play-30" % bootstrapVersion,
-    "uk.gov.hmrc" %% "play-frontend-hmrc-play-30" % "9.10.0",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30" % "1.9.0",
-    "uk.gov.hmrc" %% "emailaddress-play-30" % "4.0.0",
+    "uk.gov.hmrc" %% "play-frontend-hmrc-play-30" % "10.4.0",
```

Comment:
Better to update it to 10.5.0 as it is the latest version, if you face any issues then stay with 10.4.0